catalogs/catalog-jdbc-mysql/src/test/java/com/datastrato/gravitino/catalog/mysql/integration/test/MysqlTableOperationsIT.java

```diff
@@ -86,6 +86,7 @@ public void testOperationTable() {
         tableComment,
         properties,
         null,
+        null,
```

Comment:
Here should use `Distributions.NONE` instead of null, right? Because here is testing the behaviors after the `CatalogOperationDispatcher`, so there will be no null here

Reply:
done